x-pack/plugins/maps/public/plugin.ts

```diff
@@ -206,6 +190,7 @@ export class MapsPlugin
   public start(core: CoreStart, plugins: MapsPluginStartDependencies): MapsStartApi {
     setLicensingPluginStart(plugins.licensing);
     setStartServices(core, plugins);
+    setMapsEmsStart(plugins.mapsEms);
```

Comment:
Instead of having separate setMapsEmsStart method, how about just moving logic into setStartServices?